app/src/main/java/com/onelist/ui/Reuseable Code.kt

```diff
@@ -0,0 +1,21 @@
+package com.onelist.ui
+
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.Divider
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun categoryDivider() {
+    Divider(
+        color = MaterialTheme.colors.onBackground,
+        modifier = Modifier
+            .height(1.dp)
+            .fillMaxHeight()
+            .fillMaxWidth()
+    )
+}
```

Comment:
I did this in my project and it helped me with readability and made it easy to change certain buttons, dividers, and text. When you reuse code it helps reduce TD, try and reuse code when you find yourself writing the same thing over and over. https://arkenea.com/blog/code-review/